kernel/src/table_changes/scan.rs

```diff
@@ -254,7 +254,7 @@ mod tests {
 
         let schema = table_changes
             .schema()
-            .project(&["id", "_commit_version"])
+            .project(&["id", COMMIT_VERSION_COL_NAME])
```

Comment:
and for example I think this one is fine

Reply:
Whereas here, we're actually looking to project, we're not looking to make an assertion over our implementation.